views/package_version/_header.ejs

```diff
@@ -60,10 +73,18 @@
               </span>
             </a>
           <% } %>
-          <span class='star-count--number'><%= pkgVersion.package.stars.length %></span>
+          <span class='star-count--number'><%= pkgVersion.package.star_count %></span>
         </div>
       </div> <!-- /Do not remove this div, needed for ph-flex-position -->
 
     </div> <!-- /row -->
+    <div>
```

Comment:
I don't like where the link is added; it pushes all the content down, which is not necessary and not good SEO-wise. Also, simply putting the link in a code tag is not really elegant. Remove this for now, and don't close the issue it's related to, so we can find a better way to solve this.